app/views/new_event_mailer/notify_volunteer.html.slim

```diff
@@ -0,0 +1,8 @@
+We have just announced another event and need volunteers like you!
```

Comment:
this is a slim file, so you need to have a `p` in front to make it look like a paragraph or just `|`